tests/Forms/Prism.DryIoc.Forms.Tests/Prism.DryIoc.Forms.Tests.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
     <PackageReference Include="Moq" Version="4.13.1" />
-    <PackageReference Include="Xamarin.Forms" Version="3.6.0.344457" />
+    <PackageReference Include="Xamarin.Forms" Version="4.6.0.726" />
     <PackageReference Include="Xamarin.Forms.Mocks" Version="3.5.0.3" />
```

Comment:
The tests are currently failing for Prism.Forms. This is probably because Xamarin.Forms.Mocks will need to be updated as there is a binary break that required an update from 3.5.0.3 to 4.6.0.1... please be sure to update this